From 43bace2af6936451f6daebe2b4a55d8f85bf5ef8 Mon Sep 17 00:00:00 2001
From: Robin Burchell <robin+qt@viroteck.net>
Date: Thu, 26 Jul 2012 14:57:35 +0200
Subject: [PATCH 7/8] Revert "Fix source incompatibility error caused by QMF"

This reverts commit d83c6147c1683492afa3f92e65a0f50c1d7b59a4.

We have not yet updated QMF in Mer.
---
 src/messaging/qmessageservice_qmf.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/messaging/qmessageservice_qmf.cpp b/src/messaging/qmessageservice_qmf.cpp
index a31dd24..e50c68e 100644
--- a/src/messaging/qmessageservice_qmf.cpp
+++ b/src/messaging/qmessageservice_qmf.cpp
@@ -612,7 +612,7 @@ bool QMessageService::moveMessages(const QMessageIdList &messageIds, const QMess
     d_ptr->_active = 0;
     d_ptr->_error = QMessageManager::NoError;
     d_ptr->_active = &d_ptr->_storage;
-    d_ptr->_storage.onlineMoveMessages(convert(messageIds), convert(toFolderId));
+    d_ptr->_storage.moveMessages(convert(messageIds), convert(toFolderId));
     return true;
 }
 
-- 
1.7.7.5 (Apple Git-26)

